blog/2023-05-31-importing-data-using-axios.md

```diff
@@ -0,0 +1,218 @@
+---
+slug: importing-data-to-tolgee-using-axios
+title: 'Importing data to Tolgee using custom JS and Axios'
+authors: [ jcizmar ]
+image: '/img/blog/importing-data-to-tolgee-using-axios.png'
+description: 'Learn how to import data to Tolgee using Axios'
+tags: [ import, javascript, axios ]
+---
+
+Sometimes you might want to import the localization i18n data to Tolgee manually using the API and your custom script. In this
```

Comment:
I think that `**manually** using the API and your custom script` doesn't make much sense, I think we can remove the `manually`.

```suggestion
Sometimes you might want to import the localization i18n data to Tolgee using the API and your custom script. In this
```